packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -396,7 +400,7 @@ export class DeltaManager
             });
 
         this._inbound.on("error", (error) => {
-            this.close(CreateContainerError(error));
+            this.close(CreateContainerError(CreateCorruptionError(error)));
```

Comment:
I think CreateContainerError() wrapping should not be needed here, as error should be already of right shape, no?

Reply:
With the current implementation of `CreateContainerError`, it will definitely be a noop in the case -- happy to remove it entirely as you imply.

```suggestion
            this.close(CreateCorruptionError(error));
```

Reply:
With the current implementation of `CreateContainerError`, it will definitely be a noop in this case -- happy to remove it entirely as you imply.

```suggestion
            this.close(CreateCorruptionError(error));
```